nixos/doc/manual/release-notes/rl-2305.section.md

```diff
@@ -95,6 +95,8 @@ In addition to numerous new and upgraded packages, this release has the followin
 
 - The EC2 image module previously detected and activated swap-formatted instance store devices and partitions in stage-1 (initramfs). This behaviour has been removed. Users relying on this should provide their own implementation.
 
+- `fail2ban` has been updated to 1.0.1, which has a few breaking changes ([changelog](https://github.com/fail2ban/fail2ban/blob/1.0.2/ChangeLog#ver-101-20220927---energy-equals-mass-times-the-speed-of-light-squared))
```

Comment:
```suggestion
- `fail2ban` has been updated to 1.0.2, which has a few breaking changes ([changelog](https://github.com/fail2ban/fail2ban/blob/1.0.2/ChangeLog#ver-101-20220927---energy-equals-mass-times-the-speed-of-light-squared))
```

Reply:
The update to 1.0.1 should also be mentioned somehow since that changelog is also relevant for a double-version-jump.

Reply:
Updated it to mention all of that.